pkg/cosign/tuf/client.go

```diff
@@ -44,6 +46,10 @@ const (
 	SigstoreNoCache   = "SIGSTORE_NO_CACHE"
 )
 
+var GetRemoteRoot = func() string {
```

Comment:
Does this need to be publicly exported, or can it be a private variable in the package?

Reply:
I kind of want to make this a public global flag eventually! That would make it easy to point to preprod/staging environment. I can make it a follow up issue if you think so

As long as it's a func for now I'm fine, because I need to configure it in the tests.

Reply:
adding this to the tracking issue explicitly https://github.com/sigstore/cosign/issues/1935